pom.xml

```diff
@@ -120,5 +120,77 @@
     </profile>
   </profiles>
 
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>3.1.0</version>
+        <inherited>false</inherited>
+        <executions>
+          <execution>
+            <id>javax2jakarta</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <target>
+                <echo>${project.basedir}</echo>
+
+                <replace file="${project.basedir}/ebean-api/pom.xml"
```

Comment:
In the long run, do you think, it is a good idea to change the files in the project source dir?

I think about GIT.
- I make some changes
- I run maven
- I have a lot of changes
- It is difficult now, to sort out changes

I could imagine the following scenario:
- put "javax2jakarta" in a profile
- make also "jakarta2javax"
- before you release, convert it with "mvn -Pjavax2jakarta"
- if you have made changes, convert it back with "mvn -Pjakarta2javax"

There are changes left in the POMs. These might be done by a comment block like this
```
<!-- START javaxjakarta dependency -->
<!-- END javaxjakarta dependency -->
```
And the ant task could replace these blocks by the correct dependencies.

I know, this is more work (and maybe this should be done in a later step), but it moves the all the code/tools required to build a jakarta version in the master branch.

Reply:
> These might be done by a comment block like this

Yes. Could just enable/disable via mutating where xml comments start and end ...

> put "javax2jakarta" in a profile

Yes, I've been pondering that too and I agree it would be good to have this all in master branch.